snooty.toml

```diff
@@ -73,3 +73,19 @@ section = 1
 file = "mongotop.txt"
 title = "mongotop"
 section = 1
+
+[[banners]]
+# Warning for non-genuine deployment usage with tool binaries.
+targets = [
+        "mongodump.txt",
+        "mongorestore.txt",
+        "mongoimport.txt",
+        "mongoexport.txt",
+        "mongotop.txt",
+        "mongostat.txt",
+        "mongofiles.txt"
+        ]
+variant = "warning"
+value = """\
+        MongoDB Command Line Database Tool binaries are not supported or tested for use with non-genuine MongoDB deployments. Any usage of these tools with non-genuine MongoDB deployments may result in data corruption.
```

Comment:
Some of these tools don't modify data. Maybe instead we can say:
"Data integrity is not guaranteed when using these tools with non-genuine MongoDB deployments."

Reply:
Thinking further, maybe we can borrow from the existing language around compatibility with EOL'd versions:
"MongoDB Command Line Database Tool binaries are not supported or tested for use with non-genuine MongoDB deployments. While the tools may work on these deployments, any such compatibility is not guaranteed."

Reply:
Sure thing I updated to the latest suggestion.

Reply:
Thank you! The text looks great - I believe Abhi has shared the message about this change so let's give it a bit longer for them to respond before we push.